### PR TITLE
415 refactor integration test script for rtc s1

### PR DESCRIPTION
**Pull request by rjbdev05**
## Description
- To move the integration tests for RTC-S1 to the new paradigm, the following refactoring tasks need to be completed on the test_int_rtc_s1.sh script:

## Affected Issues
- Remove the download of the comparison script from S3 on lines 62-66
Refactor the comparison logic on lines 151-202 into a separate shell script, named compare_rtc_s1_products.sh
Refactor the comparison logic on lines 151-236, excluding lines 165-202, into a separate shell script named compare_rtc_s1_static_products.sh
Update the opera_pge_rtc_s1_delivery_5_final_runconfig.yaml runconfig to execute the new compare_rtc_s1_products.sh script within the QAExecutable section of the PGE config
Update the opera_pge_rtc_s1_static_delivery_5_final_runconfig.yaml runconfig to execute the new compare_rtc_s1_static_products.sh script within the QAExecutable section of the PGE config

The changes made in https://github.com/nasa/opera-sds-pge/pull/406 can be referenced for examples of the required changes

## Testing
- <Include any info on changes to the regression test suites (tests added, removed, modified...)>
- <Once the PR is created, don't forget to check the Build/Test Jenkins pipeline for your branch to ensure all tests are passing before merge>

### CODE REVIEW

File: .ci/scripts/rtc_s1/compare_rtc_s1_products.sh [hunk 1]
```diff
@@ -0,0 +1,77 @@
...
+
+    # add html breaks to newlines
+    compare_output=${compare_output//$'\n'/<br>$'\n'}
+    update_html_results_file "${rtc_compare_result}" "${expected_files}" "${output_files}" "${compare_output}"
+done
+
+finalize_html_results_file
+
+# Write the status code to an RC file so the integration test script can pick
+# it up.
```
Comment (JimHofman) [inline .ci/scripts/rtc_s1/compare_rtc_s1_products.sh:72]: A nit, but I see not reason to have this comment be on 2 lines

**Review by collinss-jpl — APPROVED** [review]
Integration test for RTC-S1 completes successfully, and all product comparisons pass as expected, thanks @rjbdev05!

File: .ci/scripts/rtc_s1/test_int_rtc_s1.sh [hunk 2]
```diff
@@ -90,7 +84,7 @@ fi
 echo "Creating scratch directory $scratch_dir."
 mkdir -p --mode=777 "$scratch_dir"
 
-
+expected_dir="${TMP_DIR}/${EXPECTED_DATA%.*}/"
```
Comment (JimHofman) [inline .ci/scripts/rtc_s1/test_int_rtc_s1.sh:87]: Hey Scott, in cslc_s1 weren't you able to simplify this statment, usign ${EXPECTED_DIR} or $expected_dir (sorry i forgot the exact change).  Not use if it's worth changing for consistency?

Comment (collinss-jpl) [reply .ci/scripts/rtc_s1/test_int_rtc_s1.sh:87]: It needed to be changed in the comparison script that runs within the container. For the integration test script this is the correct statement.

**Pull request merged**